packages/react-server-dom-webpack/src/ReactFlightWebpackNodeRegister.js

```diff
@@ -90,6 +98,19 @@ module.exports = function register() {
         );
       }
     }
-    return resolved;
+    // Node will now resolve again in its _load() implementation.
+    // To avoid duplicating the work, add a fast cached path.
+    Module._resolveFilename = function(r, p, i) {
```

Comment:
I guess I could override this outside and move the memoization inputs to the outer scope.

Reply:
Maybe we can assume that either it gets called or the load function exits before it and doesn't throw.

Reply:
I guess if it's reentrant that doesn't work.